Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.8 as base-python
+
+RUN apt-get clean \
+    && apt-get -y update
+
+RUN apt-get -y install nginx \
+    && apt-get -y install python3-dev \
+    && apt-get -y install libpcre3 libpcre3-dev \
+    && apt-get -y install build-essential \
+    && apt-get -y install locales \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip3 install 'poetry==1.0.9'
+
+ADD pyproject.toml poetry.lock /
+
+COPY soap_to_rest /soap_to_rest
+RUN chmod +x soap_to_rest/start.sh
```

Comment:
Does copy not pull in x permissions?  Also if you wanted to be pedantic you could do u+x here.

Reply:
Not sure. A lot of this is copied from another pattern we have.